src/components/expanded-state/AvailableNetworksv2.tsx

```diff
@@ -77,15 +78,14 @@ const AvailableNetworksv2 = ({
       const newAsset = asset;
 
       // we need to convert the mainnet asset to the selected network's
-      newAsset.mainnet_address = networks?.[ethereumUtils.getChainIdFromNetwork(Network.mainnet)]?.address ?? asset.address;
-      newAsset.address = networks?.[ethereumUtils.getChainIdFromNetwork(chosenNetwork)].address;
-      newAsset.network = chosenNetwork;
+      newAsset.mainnet_address = networks?.[chainId]?.address ?? asset.address;
```

Comment:
previously this was `Network.mainnet` for the `mainnet_address` (do you want to use `ChainId.mainnet` here?)

Reply:
oh nice good catch, going for ChainId.mainnet